datalad_debian/new_distribution.py

```diff
@@ -27,6 +27,12 @@
 @build_doc
 class NewDistribution(Interface):
     """Create a new distribution dataset
+
+    A typical distribution dataset contains a 'builder' subdataset with one or
+    more build environments, and a package subdirectory with one subdataset per
+    built Debian package.
+    This command creates the initial structure for this: A top-level dataset
```

Comment:
Small nit. I suggest removing "for this" and dropping the comma (as these are not two independent clauses).

> This command creates the initial structure: A top-level dataset under the provided path and a configured 'builder' subdataset underneath.

Reply:
thx